docs/source/torch.compiler_inductor_provenance.rst

```diff
@@ -0,0 +1,61 @@
+.. _torchinductor-provenance:
+
+TorchInductor and AOTInductor Provenance Tracking
+=================================================
+
+**Author:** `Shangdi Yu <https://github.com/yushangdi>`__ and `Rachel Guo <https://github.com/YUNQIUGUO>`__
+
+This section describes how to use the provenance tracking feature for TorchInductor and AOTInductor in ``tlparse``.
+Some example screenshots of the provenance tracking tool are shown below.
+The tool visualizes the mapping between nodes in the input graph (panel 1), the post grad graph (panel 2), and the Inductor generated code (panel 3).
+
+.. raw:: html
+
+    <style>
+        .yellow {background-color:#FFFF00;}
```

Comment:
just be mindful how this will look in the dark mode: https://docs-preview.pytorch.org/pytorch/pytorch/152211/torch.compiler_inductor_provenance.html